python/paddle/fluid/parallel_executor.py

```diff
@@ -167,9 +167,8 @@ def place_obj(place):
         # step7: init ParallelExecutor
         self.executor = core.ParallelExecutor(
             places, persistable_vars, main.desc,
-            cpt.to_text(loss_name)
-            if loss_name else six.u(''), scope, local_scopes, exec_strategy,
-            build_strategy, num_trainers, trainer_id)
+            cpt.to_text(loss_name) if loss_name else six.u(''), scope,
+            local_scopes, exec_strategy, build_strategy)
```

Comment:
Maybe add deprecation warnings when user passed `trainer_id` and `num_trainers` to parallelexecutor API

Reply:
ParallelExecutor API will be deprecated after a few PR if things go well, so no need to add warning for these 2 fields now.

Reply:
It will happen after a few PRs